examples/asset/asset_loading.rs

```diff
@@ -31,9 +31,9 @@ fn setup(
         // You might notice that this doesn't run! This is because assets load in parallel without
         // blocking. When an asset has loaded, it will appear in relevant Assets<T>
         // collection.
-        println!("{:?}", sphere.primitive_topology());
+        info!("{:?}", sphere.primitive_topology());
     } else {
-        println!("sphere hasn't loaded yet");
+        info!("sphere hasn't loaded yet");
```

Comment:
This is nitpicking but there is some inconsistency in examples now whether capitalization is used at the start for the log messages.

E.g.

> "sphere hasn't loaded yet"

vs.

> ""Custom asset loaded: {:?}" in `custom_asset.rs`

Reply:
I think starting with a capital is the right call. Feel free to follow up with another pr that makes this consistent if you feel inclined :)